cypress/support/pages/admin/pageAdministration.js

```diff
@@ -1,424 +1,473 @@
 'use strict'
 
 class Management {
```

Comment:
A lot is going on here... could you check why this is shown as diff and ensure that it still follows the code style and nothing is broken 😅